Code/Framework/AzCore/AzCore/Math/VectorN.inl

```diff
@@ -318,13 +308,23 @@ namespace AZ
         return dot;
     }
 
+    AZ_MATH_INLINE void VectorN::SetZero()
+    {
+        const AZ::Vector4 zero = AZ::Vector4::CreateZero();
+        for (Vector4& element : m_values)
+        {
+            element = zero;
+        }
```

Comment:
Again, wouldn't a memset be faster?

Reply:
Yes, this would also be faster as a memset. Good callout.